src/main/java/org/jenkinsci/plugins/prometheus/JobCollector.java

```diff
@@ -159,7 +159,7 @@ public JobCollector() {
         Jobs.forEachJob(job -> {
             logger.debug("Determining if we are already appending metrics for job [{}]", job.getName());
 
-            if (!job.isBuildable() && ignoreDisabledJobs) {
+            if (!job.isBuildable() && processDisabledJobs) {
```

Comment:
Seems the code **skips** processing if processDisabledJobs is true. It would make more sense if the variable name is **ignoreDisabledJobs**.

Reply:
@markyjackson-taulia can you please check my comment above.